pkg/grapicmd/di/providers.go

```diff
@@ -1,32 +1,39 @@
 package di
 
 import (
+	"net/http"
+
 	"github.com/google/wire"
+	"github.com/izumin5210/clig/pkg/clib"
 	"github.com/izumin5210/gex"
+	"github.com/rakyll/statik/fs"
 	"github.com/spf13/afero"
 
 	"github.com/izumin5210/grapi/pkg/cli"
 	"github.com/izumin5210/grapi/pkg/excmd"
+	"github.com/izumin5210/grapi/pkg/gencmd"
 	"github.com/izumin5210/grapi/pkg/grapicmd"
 	"github.com/izumin5210/grapi/pkg/grapicmd/internal/module"
-	"github.com/izumin5210/grapi/pkg/grapicmd/internal/module/generator"
 	"github.com/izumin5210/grapi/pkg/grapicmd/internal/module/script"
 	"github.com/izumin5210/grapi/pkg/grapicmd/internal/usecase"
 	"github.com/izumin5210/grapi/pkg/protoc"
 )
 
-func ProvideGenerator(ctx *grapicmd.Ctx, ui cli.UI) module.Generator {
-	return generator.New(
-		ctx.FS,
-		ui,
-	)
-}
-
 func ProvideScriptLoader(ctx *grapicmd.Ctx, executor excmd.Executor) module.ScriptLoader {
 	return script.NewLoader(ctx.FS, executor, ctx.RootDir.String())
 }
 
-func ProvideInitializeProjectUsecase(ctx *grapicmd.Ctx, gexCfg *gex.Config, ui cli.UI, fs afero.Fs, generator module.Generator, excmd excmd.Executor) usecase.InitializeProjectUsecase {
+func ProvideGenerator(ctx *grapicmd.Ctx, ui cli.UI, fs afero.Fs, tmplFs http.FileSystem, outDir clib.Path) gencmd.Generator {
+	return gencmd.NewGenerator(
+		fs,
+		ui,
+		outDir,
+		tmplFs,
+		nil,
+	)
+}
+
+func ProvideInitializeProjectUsecase(ctx *grapicmd.Ctx, gexCfg *gex.Config, ui cli.UI, fs afero.Fs, generator gencmd.Generator, excmd excmd.Executor) usecase.InitializeProjectUsecase {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
line is 184 characters (lll)